frontend2/src/components/tables/submissions/SubHistoryTable.tsx

```diff
@@ -100,7 +111,10 @@ const SubHistoryTable: React.FC<SubHistoryTableProps> = ({
         {
           header: "",
           key: "download",
-          value: (sub) => "Download",
+          value: (sub) => <button className="text-cyan-600 hover:underline"
```

Comment:
nit: I think we have a `Button` component that we should prefer to use for consistent styling